isomerpages/isomercms-frontend
# chore: update snyk

## Problem
need to update packages to satisfy snyk security warnings

## Solution
https://www.notion.so/opengov/Snyk-reviews-252f290d93524e7e916503719f04e78d?pvs=4#31dbbfea17b841d299c6063962b7c4ca

As much as possible, i have tried to install **minor** versions but we should check functionalty

## Tests
- Go to any site 
- make an edit
- save
- the preview should be accurate 
- run `npm run build` -> the repo should build
- [post staging push] 
- `datadog` should stilll have frontend code traces
